docs/guides/advanced/using-environments.md

```diff
@@ -0,0 +1,48 @@
+If you set up Logfire according to the [getting started guide](../../index.md), you already have instrumented logfire in your applications
+via the `logfire.configure()` function. If you find yourself wanting to distinguish between data sent in one environment (e.g. `dev` vs. `production`), it may be overkill to create a new project for each (more on how to decide this below).
+
+Instead you can use the environments feature, which is a special kind of resource attribute applied to the whole payload received by Logfire.
+This attribute says which environment the payload comes from.
+
+You can set a Logfire environment via the SDK `logfire.configure` function:
+
+```py title="main.py"
+import logfire
+
+logfire.configure(environment='dev')
+
+from app.main import main
+
+main()
+```
+
+Under the hood, this sets the OTel [`deployment.environment.name`](https://opentelemetry.io/docs/specs/semconv/resource/deployment-environment/).
+Note that you can also set this via the `LOGFIRE_ENVIRONMENT` environment variable.
```

Comment:
I think more important than `LOGFIRE_ENVIRONMENT` would be to explain how to set this in other languages... e.g. with the `OTEL_RESOURCE_ATTRIBUTES="deployment.environment.name=prod"` environment variable.

Reply:
Thanks, added